app/components/pipeline/schedules/Form.js

```diff
@@ -30,50 +30,50 @@ class Form extends React.Component {
     return (
       <div>
         <FormTextField
+          label="Description"
+          help="The description for the schedule (supports :emoji:)"
+          required={true}
+          errors={errors.findForField("label")}
+          value={this.props.label}
+          ref={(labelTextField) => this.labelTextField = labelTextField}
+        />
+
+        <FormTextField
           label="Cron Interval"
-          help="The interval or time that this schedule should run using cron syntax, e.g (<code>30 * * * *</code> or <code>@midnight</code>). These times should be in UTC."
+          help={"The interval for when builds will be created, in UTC, using <a class=\"lime\" target=\"_blank\" rel=\"noopener\" href=\"https://crontab.guru\">cron format</a>. Also supports <code>@monthly</code>, <code>@weekly</code>, <code>@daily</code>, <code>@midnight</code>, and <code>@hourly</code>. For example, <code>30 * * * *</code> creates a build on the 30th minute of every hour."}
+          required={true}
           errors={errors.findForField("cronline")}
           value={this.props.cronline}
           ref={(cronlineTextField) => this.cronlineTextField = cronlineTextField}
         />
 
         <FormTextField
-          label="Label"
-          help="Describe what this schedule is all about (you can even use :emoji:)"
-          errors={errors.findForField("label")}
-          value={this.props.label}
-          ref={(labelTextField) => this.labelTextField = labelTextField}
-        />
-
-        <FormTextField
           label="Message"
-          help="The message to use for the created build"
+          help="The message to use for the build. Default: “Scheduled Build for (Pipeline Name)”"
           errors={errors.findForField("message")}
           value={this.props.message}
           ref={(messageTextField) => this.messageTextField = messageTextField}
         />
 
         <FormTextField
           label="Commit"
-          help="The commit to use for the created build"
+          help="The commit to use for the build. Default: “HEAD”"
           errors={errors.findForField("commit")}
           value={this.props.commit}
-          placeholder={"HEAD"}
           ref={(commitTextField) => this.commitTextField = commitTextField}
         />
 
         <FormTextField
           label="Branch"
-          help="The branch to use for the created build"
+          help={`The branch to use for the build. Default: the pipeline’s default branch (currently “${this.props.pipeline.defaultBranch}”)`}
```

Comment:
Niiiice 👌🏻 What do you think about saying 'yours is currently __'? Otherwise people might think this help text is generic?